game/src/sandbox/dashboards/travel_times.rs

```diff
@@ -32,17 +33,7 @@ impl TravelTimes {
                 filter.modes.contains(&mode),
             ));
         }
-        filters.push(Widget::dropdown(
```

Comment:
This particular filter applies only to the top two visualizations. Similarly, the "include unaffected" checkbox only applies to the "delays" matrix, so I split the filter controls up. WDYT?

Reply:
Splitting like this makes sense